presto-main/src/test/java/com/facebook/presto/operator/spiller/BenchmarkBinaryFileSpiller.java

```diff
@@ -142,6 +142,7 @@ public void tearDown()
                     DOUBLE.writeDouble(pageBuilder.getBlockBuilder(4), lineItem.getExtendedPrice());
                 }
                 pages.add(pageBuilder.build());
+                pageBuilder.reset();
```

Comment:
If your intention indeed is to fix this single benchmark, please make it explicit in commit message and PR title